overlord/configstate/hooks.go

```diff
@@ -73,7 +74,8 @@ func (h *configureHandler) Before() error {
 	h.context.Lock()
 	defer h.context.Unlock()
 
-	tr := ContextTransaction(h.context)
+	task, _ := h.context.Task()
+	rt := configcore.NewRunTransaction(ContextTransaction(h.context), task)
```

Comment:
do we need this change now?

Reply:
As the transaction here is used only in the call to Patch(), I think we can leave it as it is, so I've reverted here.